OpenMEEG/src/assemble.cpp

```diff
@@ -50,27 +52,30 @@ knowledge of the CeCILL-B license and that you accept its terms.
 
 using namespace std;
 using namespace OpenMEEG;
+using Strings = vector<string>;
 
 unsigned gauss_order = 3;
 
 void getHelp(char** argv);
 
+bool option(const string& option, const Strings& options);
+
 int main(int argc, char** argv)
 {
     print_version(argv[0]);
 
     bool OLD_ORDERING = false;
-    if ( argc<2) {
-        cerr << "Not enough arguments \nPlease try \"" << argv[0] << " -h\" or \"" << argv[0] << " --help \" \n" << endl;
+    if ( argc<2 ) {
+        getHelp(argv);
         return 0;
```

Comment:
instead of complaining: help the user. (do we go for that behavior in other binarys ?)